docs/02-app/01-building-your-application/02-data-fetching/01-fetching-caching-and-revalidating.mdx

```diff
@@ -211,28 +211,28 @@ You can also use the experimental [`unstable_cache` API](/docs/app/api-reference
 In the example below:
 
 - The `revalidate` option is set to `3600`, meaning the data will be cached and revalidated at most every hour.
-- The React `cache` function is used to [memoize](/docs/app/building-your-application/caching#request-memoization) data requests.
+- The Next.js `unstable_cache` function is used to [memoize](/docs/app/building-your-application/caching#request-memoization) data requests.
```

Comment:
This needs to be updated

Reply:
Does something like `The Next.js `unstable_cache` function is used to memoize expensive operations, like database queries, and reuse them across multiple requests`, combining this and the docs from `unstable_cache`?